### ISSUE TITLE
How long did it take to train the model?

### ISSUE THREAD

**Issue by liming-ai** [bfshi/DGAM-Weakly-Supervised-Action-Localization#12]
Hi @bfshi , thanks for your contribution, you reported the whole model is trained on single NVIDIA Tesla M40 GPU, could you please tell me how long did it take to train the model on ActivityNet?

**Comment by bfshi** [comment]
Hi @mitming, I think it took about half a day.

**Comment by liming-ai** [comment]
> Hi @mitming, I think it took about half a day.

Thank you very much! Could you please tell me whether 3 NVIDIA 1080Ti GPUs can meet the need of action detection?

**Comment by bfshi** [comment]
Do you mean training DGAM? Yes I think it's enough because 1080Ti is faster than M40. But our repo uses features extracted beforehand. If you need to train a model on raw frames, then I think 3 cards may not be enough for efficient training.

**Comment by liming-ai** [comment]
> Do you mean training DGAM? Yes I think it's enough because 1080Ti is faster than M40. But our repo uses features extracted beforehand. If you need to train a model on raw frames, then I think 3 cards may not be enough for efficient training.

Actually I am a freshman for action detection, I tried some action recognition datasets but found I could not training them.

I don’t know if the SOTA methods in action detection all use publicly available pre-trained models? Or I have to train a model on raw frames for better accuracy?

**Comment by bfshi** [comment]
Recent action detection (fully/weakly-supervised) papers most use pretrained feature instead of directly training on raw frames, so I think 3 1080Ti should be enough?

**Comment by liming-ai** [comment]
> Recent action detection (fully/weakly-supervised) papers most use pretrained feature instead of directly training on raw frames, so I think 3 1080Ti should be enough?

Thank you very much, you give me confidence!!! Best wish to you!

**Comment by leftthomas** [comment]
@bfshi the total epochs for  `ActivityNet 1.2` seems too large, and it takes about one and half days on single TITAN X GPU
https://github.com/bfshi/DGAM-Weakly-Supervised-Action-Localization/blob/752bd457156fda8fefb96f8c71d55141cd4e39ef/lib/core/config.py#L114
